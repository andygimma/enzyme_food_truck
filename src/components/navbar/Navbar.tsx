import { FormattedMessage } from "react-intl";
import styled from "styled-components";
import "@fontsource/caveat"; // Defaults to weight 400.

const Container = styled.div`
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  color: white;
  /* background-color: #7e2e84; */
  background: rgb(2, 0, 36);
  background: linear-gradient(
    90deg,
    rgba(2, 0, 36, 1) 0%,
    rgba(126, 46, 132, 1) 50%,
    rgba(2, 0, 36, 1) 100%
  );
  height: 60px;
`;

const Header = styled.h1`
  font-size: 48px;
  font-family: "Caveat";
`;

const Navbar = () => {
  return (
    <Container>
      <Header>
        <FormattedMessage id="app.company_name" defaultMessage={"Food Truck"} />
      </Header>
    </Container>
  );
};

export default Navbar;
