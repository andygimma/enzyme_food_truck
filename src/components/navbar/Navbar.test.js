import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import Navbar from "./Navbar";

describe("Navbar", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<IntlProvider locale="en"><Navbar /></IntlProvider>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})

