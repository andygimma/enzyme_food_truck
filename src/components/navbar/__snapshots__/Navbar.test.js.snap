// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  className="sc-bczRLJ jFJXRG"
>
  <h1
    className="sc-gsnTZi cLmPLR"
  >
    Food Truck
  </h1>
</div>
`;
